tamr_unify_client/models/project/resource.py

```diff
@@ -120,6 +123,16 @@ def input_datasets(self):
         alias = self.api_path + "/inputDatasets"
         return DatasetCollection(self.client, alias)
 
+    def attribute_configurations(self):
+        """ Project's attribute's configurations.
+        :returns: the configurations of the attributes of a project
+        :rtype :class: 'tamr_unify_client.models.attribute_configuration.collection.AttributeConfigurationCollection'
```

Comment:
To render in docs should be :class:\`~tamr_unify_client.models.attribute_configuration.collection.AttributeConfigurationCollection\`